### ISSUE TITLE
After Update to 1.4.6 node Openhab2-in2 do not work

### ISSUE THREAD

**Issue by mispezi** [jeroenhendricksen/node-red-contrib-openhab3#48]
after updating and putting in  an openhab2-in2-node this get only an update by deploying it.
My workaround in the moment is using of openhab2-events2 with filters for items I want to get updated.
This node works correctly.

**Comment by mispezi** [comment]
Yes, I am shure. I only use one configuration. The thing I don't anderstand is that the deploying of the node produces values and an update of values on the OH-side is ignorred. But my work-around works for me. So let us see if there are others with the same problem.

**Comment by JoeRu** [comment]
Hi - i am sort of hagling the same issue. It is like not every update is pushed through to openhab. It took me a while to figure this out...

**Comment by JoeRu** [comment]
> Hi - i am sort of hagling the same issue. It is like not every update is pushed through to openhab. It took me a while to figure this out...

Well - turned out - i'm wrong - this error seems to be about the "old" openhab2 plugin which is no longer maintained. Yours is working quite fine... Sorry ...
